gapic/api/artman_bigtable.yaml

```diff
@@ -6,46 +6,65 @@ common:
   proto_deps:
     - google-common-protos
   import_proto_path:
-    - ${REPOROOT}/googleapis
+    - ${GOOGLEAPIS}
   src_proto_path:
-    - ${REPOROOT}/googleapis/google/bigtable/v2
+    - ${GOOGLEAPIS}/google/bigtable/v2
   service_yaml:
-    - ${REPOROOT}/googleapis/google/bigtable/bigtable.yaml
+    - ${GOOGLEAPIS}/google/bigtable/bigtable.yaml
   output_dir: ${REPOROOT}/artman/output
+  git_repos:
+    staging:
+      location: git@github.com:googleapis/api-client-staging.git
   gapic_api_yaml:
-    - ${REPOROOT}/googleapis/google/bigtable/v2/bigtable_gapic.yaml
+    - ${GOOGLEAPIS}/google/bigtable/v2/bigtable_gapic.yaml
 csharp:
   gapic_code_dir: ${REPOROOT}/artman/output/gcloud-csharp-cloud-bigtable
 go:
   gapic_code_dir: ${REPOROOT}/gapi-cloud-bigtable-go
-  git_repo:
-    location: git@github.com:GoogleCloudPlatform/google-cloud-go.git
-    gapic_subpath: bigtable/apiv2
+  git_repos:
+    go:
+      location: git@github.com:GoogleCloudPlatform/google-cloud-go.git
+      gapic_subpath: bigtable/apiv2
+    staging:
+      gapic_subpath: google-cloud-bigtable-v2/cloud.google.com/go/bigtable/apiv2
 java:
-  gapic_code_dir: ${REPOROOT}/google-cloud-java/google-cloud-bigtable
-  git_repo:
-    gapic_subpath: generated/java/google-cloud-bigtable-v2
+  gapic_code_dir: ${REPOROOT}/artman/output/java/google-cloud-bigtable
+  git_repos:
+    java:
+      gapic_component: src/main/java/com/google/cloud/bigtable/spi/v2
+      gapic_subpath: google-cloud-pubsub/src/main/java/com/google/cloud/bigtable/spi/v2
```

Comment:
> In the google-cloud repo, the existing code is not removed before new code is copied in, so there is no problem

`--staging=local` very much removes what used to be at that spot ([here][1]). This is also true of the Artman [_status quo ante_][2] before my refactor last week.

  [1]: https://github.com/googleapis/artman/blob/master/artman/tasks/publish/local.py#L59
  [2]: https://github.com/googleapis/artman/blob/e45376ffd4eb6d3d9f9f3bdac5706cea9adf4d70/pipeline/tasks/gapic_tasks.py#L120

That said, my feeling on the language specific segments of these files is that the language owners decide. Therefore, I will have this changed to what you are asking momentarily. If it turns out it is not right for you, you are always free to change it (and you very much do not need my buy-in to do so). :-)